incubator/hnc/pkg/kubectl/configset.go

```diff
@@ -0,0 +1,67 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+var setTypeCmd = &cobra.Command{
+	Use: fmt.Sprintf("set-type --apiVersion X --kind Y <%s|%s|%s>",
+		api.Propagate, api.Remove, api.Ignore),
+	Short: "Sets the HNC configuration of a specific type",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
If you type in `kubectl help create role`, you get a lot of examples. Can we add some examples here as well? I don't know if they belong in the "Use" field or somewhere else (e.g., is ther a "Long" help field?).

Reply:
Sure! I added two examples in the [Example](https://github.com/spf13/cobra/blob/138b98f39dedc35ae01b827fb9bceacaf30b0880/command.go#L56) field and removed the examples in the flag definitions. Feel free to take a look.